defaults/main.yml

```diff
@@ -1 +1,17 @@
 # defaults file for reboot
+
+# seconds to wait before reboot to allow ansible to wrap up it's connection
+reboot_seconds_before_reboot: "30"
```

Comment:
Do we want the quotes? Do we want to force these three variables to be strings instead of integers?